sklearn/tests/test_docstring_parameters.py

```diff
@@ -190,21 +216,8 @@ def test_fit_docstring_attributes(name, Estimator):
     doc = docscrape.ClassDoc(Estimator)
     attributes = doc['Attributes']
 
-    IGNORED = {
-        'ClassifierChain',
-        'CountVectorizer', 'DictVectorizer',
-        'GaussianRandomProjection',
-        'MultiOutputClassifier', 'MultiOutputRegressor',
-        'NoSampleWeightWrapper', 'RFE', 'RFECV',
-        'RegressorChain', 'SelectFromModel',
-        'SparseCoder', 'SparseRandomProjection',
-        'SpectralBiclustering', 'StackingClassifier',
-        'StackingRegressor', 'TfidfVectorizer', 'VotingClassifier',
-        'VotingRegressor', 'SequentialFeatureSelector',
-    }
-
-    if Estimator.__name__ in IGNORED or Estimator.__name__.startswith('_'):
-        pytest.skip("Estimator cannot be fit easily to test fit attributes")
+    if Estimator.__name__.startswith("_"):
+        pytest.skip("Do not test private estimators")
```

Comment:
I think this is useless since `all_estimators` already filters out private estimators